velox/common/base/Exceptions.h

```diff
@@ -312,6 +312,7 @@ DECLARE_CHECK_FAIL_TEMPLATES(::facebook::velox::VeloxRuntimeError);
 #define VELOX_DCHECK_LE(e1, e2, ...) VELOX_CHECK(true)
 #define VELOX_DCHECK_EQ(e1, e2, ...) VELOX_CHECK(true)
 #define VELOX_DCHECK_NE(e1, e2, ...) VELOX_CHECK(true)
+#define VELOX_DCHECK_NULL(e, ...) VELOX_CHECK(true)
```

Comment:
Nice fix. Would you extract it into a separate PR?

Reply:
will do

Reply:
Opened https://github.com/facebookincubator/velox/pull/8644